### ISSUE TITLE
`RtApiAlsa::startStream()`, `stopStream()`, and `abortStream()` return RTAUDIO_SYSTEM_ERROR on success

### ISSUE THREAD

**Issue by AndrewBelt** [thestk/rtaudio#342]
I believe this is caused by a typo in each of these functions when checking the `result` variable. ALSA functions return negative on error and 0 on success. `if (result >= 0)` should be `if (result < 0)`. e.g. https://github.com/thestk/rtaudio/blob/f9fd8726ccb688baa43aaae642179a69d353f6d2/RtAudio.cpp#L8280

**Comment by garyscavone** [comment]
Yes, that is a bug. I fixed it in the newdeviceselection branch but not in master. If someone has time to do it, please make a PR.

**Comment by garyscavone** [comment]
I just fixed this.
